### ISSUE TITLE
Can I group widgets and enable/disable them by only one operation?

### ISSUE THREAD

**Issue by occoder** [dddomodossola/remi#439]
I got quite a few widgets (e.g. spin) that need to be enabled or disabled in sync.
Should I have to enable/disable individual widget one by one? Or maybe I can group them and just operate once on such group?
Thanks in advance.

**Comment by dddomodossola** [comment]
Heĺlo @occoder , it is ok to disable groups of widgets ;-)

**Comment by occoder** [comment]
> Heĺlo @occoder , it is ok to disable groups of widgets ;-)

Should I wrap all widgets into a container and operate on that container?

**Comment by dddomodossola** [comment]
@occoder yes it is simpler to do this way

**Comment by occoder** [comment]
> @occoder yes it is simpler to do this way

@dddomodossola I did wrap these widgets into a HBox container and tried to enable/disable the HBox object. But it turned out that the individual spin widget is NOT grayed out if its parent HBox container is disabled. I don't what could possibly go wrong?

**Comment by dddomodossola** [comment]
Which remi version are you using? I suggest you to update to the latest version and refresh the browser cache.

**Comment by occoder** [comment]
@dddomodossola I used the latest version, i.e. 2021.3.2. I found that xxx_hbox.set_enabled(False) does not gray out the HBox container. And this #373 indicates that container could (only?) be disabled through a workaround method like 
```
mycontainer.style['pointer-events'] = 'none'
mycontainer.style['opacity'] = '0.4' #this is to give a disabled apparence
```
and re-enabled through
`del mycontainer.style['pointer-events']`
Is this still the situation in the latest version of Remi?
